syntax = "proto3";

message PWMValues {
    uint32 ch1 = 1; // Rudder
    uint32 ch2 = 2; // Movable ballast
    uint32 ch3 = 3; // Trim tab state setting 1
    uint32 ch4 = 4; // Trim tab state setting 2
    uint32 ch5 = 5; // Trim tab manual angle
    uint32 ch6 = 6; //
}

message ControlAngles {
    uint32 rudder_angle = 1;
    uint32 ballast_angle = 2;
}